setup.py

```diff
@@ -118,7 +108,8 @@ def build_extension(self, ext: CMakeExtension):
     "url": "https://github.com/XanaduAI/pennylane-lightning",
     "license": "Apache License 2.0",
     "packages": find_packages(where="."),
-    "package_data": {"pennylane_lightning": ["src/*"]},
+    "package_data": {"pennylane_lightning": ["src/*", "src/**/*"]},
```

Comment:
Hi @mlxd, I just change this line. All other changes are from `black`.  You can check wheels here: https://github.com/PennyLaneAI/pennylane-lightning/actions/runs/1796752045. If it looks good, I will revert workflow file and merge.